podzim_2025/DagyDee/bash-1.txt

```diff
@@ -0,0 +1,70 @@
+Úkoly - Bash I
+
+1. Co dělá příkaz ls -v?
+    
+    C. Seřadí soubory podle hodnot (víceciferných) čísel v názvech
+    
+    A. ls -1 --version  # Vypíše verzi programu ls
+    B. ls -d            # Vypíše jméno aktuálního adresáře, ne jeho obsah
+    D. ls -m  	        # Vypíše obsah adresáře kompaktně – jednotlivá jména oddělí čárkou
+
+2. Začnu-li (jako uživatel mh) v adresáři /home/mh/data, které z těchto příkazů mě dostanou do /home/mh?
+
+    C. cd /home/mh
+    E. cd ~
+    G. cd ~/data/..
+    H. cd
+    I. cd ..
+
+3. Mám-li níže ukázané adresáře, a pwd vypisuje /home/mh/vec, co ukáže ls -F ../archiv?
+
+    E. original/ pnas_final/ pnas_2/
+
+4. Vypadá-li disk jako v předchozím úkolu a pwd vypisuje /home/mh/archiv, které z těchto příkazů vypíšou pnas_2/ pnas_final/ original/?
+
+    B. ls -r -F
+    C. ls -r -F /home/mh/archiv
+
+5. Zadej tenhle příkaz: touch novy_soubor.txt
+
+    Co tenhle příkaz udělal? (Nápověda: Použij ls)  -> vytvořil soubor "novy_soubor.txt"
+    Když se do aktuálního adresáře podíváš grafickým manažerem souborů, ukáže se ti novy_soubor.txt?  -> ano
+    Pusť ls -l. Zjistíš z výpisu, co nový soubor obsahuje?  -> ne
+    K čemu slouží příkaz touch?  -> k vytvoření nového souboru
+
+6. Po tom, co pustíš následující příkazy, si uvědomíš, že soubory sucrose.dat a maltose.dat jsou špatně umístěné – měly by být v adresáři raw.
+
+	mv sucrose.dat maltose.dat ../raw
+
+7. Vytvořila sis soubor statsitika.txt, ve kterém máš nějaké výpočty. Ale ve jméně je překlep! Které z těchto příkazů to spraví?
+
+   A. cp statsitika.txt statistika.txt  -> budu mít opravenou kopii, původní soubor zůstane
+   B. mv statsitika.txt statistika.txt
+
+8. Co vypíše ls na konci této sekvence příkazů?
+
+    B. rekombinace
+
+9. Zkus napsat co nejkratší příkaz, kterým přesuneš fructose.dat a sucrose.dat do adresáře analyzed/.
+
+	mv fructose.dat sucrose.dat analyzed
+
+10. Které z následujících sad příkazů vytvoří tuhle strukturu adresářů?
```

Comment:
Zkus to ještě jednou, A a D máš dobře, E bohužel správně není. Ale je tam ještě jedna správná možnost.